src/core/Microsoft.PowerToys.Settings.UI.Lib/PowerLauncherProperties.cs

```diff
@@ -46,7 +46,7 @@ public class PowerLauncherProperties
 
         public PowerLauncherProperties()
         {
-            OpenPowerLauncher = new HotkeySettings();
+            OpenPowerLauncher = new HotkeySettings(false, false, true, false, 32);
```

Comment:
[nit] we could use `Windows.System.VirtualKey.Space` instead of hardcoding the value.

Reply:
this seems to be all over the place :) some initializations are using a number in a hex format. Lets unify this! 
Do we want to use VirtualKey enum values everywhere? We could actually have VirtualKey Key property in HotkeySettings which is not serialized just returns int Code typed to that enum

Reply:
Yes, we could do that. I think the problem with the [VirtualKey enum](https://docs.microsoft.com/en-us/uwp/api/Windows.System.VirtualKey?view=winrt-19041) might be that it doesn't have all virtual key codes listed from 0-255 that are documented in the [virtual key codes doc](https://docs.microsoft.com/en-us/windows/win32/inputdev/virtual-key-codes), so I don't know if switching to it will work for all scenarios. We could create a separate issue to look into that, but I don't think that needs to be done within this PR.